eventhandling/configureEvent.go

```diff
@@ -370,7 +371,15 @@ func createScrapeJobConfig(scrapeConfig *prometheus.ScrapeConfig, config *promet
 	// define scrape job name
 	scrapeConfig.JobName = scrapeConfigName
 	// set scrape interval to 5 seconds
-	scrapeConfig.ScrapeInterval = prometheus_model.Duration(5 * time.Second)
+	scrapeInterval := utils.EnvVarOrDefault("SCRAPE_INTERVAL", "5")
+	scrapeIntervalInt, err := strconv.Atoi(scrapeInterval)
+
+	if err != nil {
+		scrapeConfig.ScrapeInterval = prometheus_model.Duration(5 * time.Second)
```

Comment:
I think we could add a logging output here to help the user troubleshoot a malformed `SCRAPE_INTERVAL` value, otherwise if an parsing error happens the scrape config will silently use the default value.

Reply:
Yes, that seems like a good idea. It should be implemented now.